src/Opdex.Platform.WebApi/IndexerBackgroundService.cs

```diff
@@ -141,16 +148,16 @@ public override async Task StopAsync(CancellationToken cancellationToken)
         }
         catch (Exception ex)
         {
-            _logger.LogCritical(ex, "Failure gracefully shutting down the indexer, indexing locked");
+            _logger.LogCritical(ex, "Failure gracefully shutting down the indexer, indexing locked.");
         }
         finally
         {
-            await base.StopAsync(CancellationToken.None);
+            await base.StopAsync(cancellationToken);
         }
     }
-
-    public override Task StartAsync(CancellationToken cancellationToken)
-    {
-        return base.StartAsync(cancellationToken);
-    }
 }
+
+public class IndexerConfiguration
+{
+    public bool Enabled { get; set; }
+}
```

Comment:
We should continue to make separate file/classes for configurations so things don't get lost within other fies.

Reply:
In this case we need to use `IOptionsMonitor` and so I think it's best to keep separate, so we're not monitoring config which doesn't need to be monitored. I'll move into a separate file